resources/views/blueprints/index.blade.php

```diff
@@ -133,6 +133,24 @@
     </div>
     @endforeach
 
+    @foreach (Statamic\Facades\Blueprint::additionalBlueprints() as $title => $blueprints)
+        <h3 class="little-heading pl-0 mb-1">{{ $title }}</h3>
+        <div class="card p-0 mb-2">
+            <table class="data-table">
+                @foreach ($blueprints as $blueprint)
+                    <tr>
+                        <td>
+                            <div class="flex items-center">
+                                <div class="w-4 h-4 mr-2">@cp_svg('drawer-file')</div>
```

Comment:
I wonder if it might be better to just use a generic icon here, rather than the forms one. (maybe even a way to specify a custom icon)

Reply:
Oh yeah good catch. A generic `@cp_svg('blueprint')` icon would be fine for now.

Reply:
Yeah, i had the same idea, but thought that the amount of code needed to add that feature wasn't worth it for a first release. Pushed an icon chagne